qa/tasks/cephfs/fuse_mount.py

```diff
@@ -142,13 +153,8 @@ def _add_valgrind_args(self, mount_cmd):
     def _list_fuse_conns(self):
         conn_dir = "/sys/fs/fuse/connections"
 
-        self.client_remote.run(args=['sudo', 'modprobe', 'fuse'],
-                               check_status=False)
-
         if not self.client_remote.is_mounted(conn_dir):
-            self.client_remote.run(
-                args=["sudo", "mount", "-t", "fusectl", conn_dir, conn_dir],
-                timeout=30, omit_sudo=False)
```

Comment:
nit: Did we want to preserve the `timeout=30`? Not specifying timeout will default to 300.

Otherwise, LGTM

Reply:
I don't think we need to preserve that timeout. These values were all arbitrarily chosen.

Reply:
300 seconds is too long. It causes vstart_runner and teuthology tests to be stuck for 5 minutes unnecessarily. To prevent this, IIRC, Zheng and I had decided to change these timeouts to 30 seconds. So, it would be useful if we preserve it to 30 seconds.

Reply:
I looked at #34839 and didn't see any discussion WRT timeouts.

FWIW, these paramiko timeouts never worked as expected so I doubt that change did anything at all functional. It is because those timeouts are all but useless that this PR (and stdin-killer) exists.